docs/docs/documentation/getting-started/installation/postgres.md

```diff
@@ -21,9 +23,9 @@ services:
     ports:
       - "9925:3000" # (2)
     volumes:
-      - mealie-data:/app/data/ # (3)
+      - ./mealie-data:/app/data/ # (3)
```

Comment:
this is a volume name not a path

Reply:
This change means that the volume is created inside the same location as the mealie docket compose. Otherwise the volume lives in the weird docker volume default and can be hard for users to find.

Reply:
> My thought was that doing this would make it easier to troubleshoot questions/bugs, since directing users to logs, db files, etc is all local to the same directory as the docker compose file.

> But happy to go back if obfuscating the mealie data dir inside docker’s default volume location is your preferred behavior.

Could you do that in another PR and we can get some feedback from the community? My guess is their preference would be what you've done, even though it's not mine. But I'd like to get some feedback before we change it.

Reply:
Will do. Do you prefer getting feedback via GitHub (e.g. a discussion) or docker?  In the meantime, I’ll put the docker composes back to what they originally were (in an hour or so).